model/actions/GetActiveDeliveryExecution.php

```diff
@@ -77,14 +78,14 @@ public function __invoke($params)
 
             /** @var AttemptServiceInterface $attemptService */
             $attemptService = $this->getServiceLocator()->get(AttemptServiceInterface::SERVICE_ID);
-            $satesToExclude = $attemptService->getStatesToExclude();
+            $statesToExclude = $attemptService->getStatesToExclude();
             //filter sates which should not be treated as an attempt
-            $executions = array_filter($executions, function ($execution) use ($satesToExclude) {
-                return !in_array($execution->getState()->getUri(), $satesToExclude);
+            $executions = array_filter($executions, function ($execution) use ($statesToExclude) {
+                return !in_array($execution->getState()->getUri(), $statesToExclude);
             });
```

Comment:
Just an idea: we can use method extract refactoring here move this block into a dedicated method to enhance readability:

```
private function filterExecutionsByState(array $executions): array
{
    /** @var AttemptServiceInterface $attemptService */
    $attemptService = $this->getServiceLocator()->get(AttemptServiceInterface::SERVICE_ID);
    $statesToExclude = $attemptService->getStatesToExclude();
            
    return array_filter($executions, function ($execution) use ($statesToExclude) {
        return !in_array($execution->getState()->getUri(), $statesToExclude);
    });
}
```

Then:
```
$executions = $this->filterExecutionsByState($executions);
```